core/src/main/java/edu/wpi/grip/core/operations/Operations.java

```diff
@@ -15,6 +15,7 @@ private Operations() { /* no op */}
 
     public static void addOperations(EventBus eventBus) {
         // Add the default built-in operations to the palette
+        eventBus.post(new OperationAddedEvent(new ResizeOperation()));
```

Comment:
What is the sort order of the operations? It should have some set order so people can expect where the operations are in the list.